sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -16,6 +27,9 @@
 
 #include <_az_cfg_prefix.h>
 
+/**
+ * @brief Client is fixed to a specific version of the Azure Storage Blobs service
```

Comment:
```suggestion
 * @brief Client is fixed to a specific version of the Azure Blob Storage service.
```

Reply:
For now I'm going to stay with what's in the code for consistency.  We can revisit for the next release